src/utils.js

```diff
@@ -61,8 +61,8 @@ const fetchUploadData = async (opts) => {
 };
 
 const upload = async (opts) => {
-  const { path } = opts;
-  const endpoint = 'https://asset.cml.dev';
+  const { path, domain = 'asset.cml.dev' } = opts;
```

Comment:
```suggestion
  const { path, domain } = opts;
```

Consider setting the default on the `yargs` options instead.

Reply:
the downside of not default for unpacking: https://github.com/iterative/cml/runs/7014485551?check_suite_focus=true#step:7:81

Reply:
Valid concern, although keeping defaults in two different places (command-line interface and underlying library) doesn't sound quite optimal to me. Would you prefer to put back the unpacking default value, update the tests instead, something else?

Reply:
without pushing your `utils` button, I originally went with the unpacking default since it was in the utils module.

Reply:
My `utils` button is [vandal-ressistant](https://en.wikipedia.org/wiki/Vandal-resistant_switch); feel free to restore the unpacking default. 😄